build.fsx

```diff
@@ -89,6 +89,7 @@ let activeProjects =
 
     !! "src/**/*.??proj"
     ++ "tests/**/OpenTK.Tests.fsproj"
+    ++ "tests/**/OpenTK.Tests.Integration.fsproj"
```

Comment:
If we add more tests in the future, perhaps better to have generic rule? OpenTK.Tests*fsproj